docs/src/api/class-locatorassertions.md

```diff
@@ -1523,6 +1523,12 @@ Expected class or RegExp or a list of those.
 
 Expected class or RegExp or a list of those.
 
+### option: LocatorAssertions.toHaveClass.partial
```

Comment:
A topic for the API review: does `not.toHaveClass('foo bar', { partial: true })` match the class name `foo baz`?